exchanges/bithumb/bithumb_websocket.go

```diff
@@ -0,0 +1,213 @@
+package bithumb
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+)
+
+const wsEndpoint = "wss://pubwss.bithumb.com/pub/ws"
+
+var (
+	wsDefaultTickTypes = []string{"30M"} // alternatives "1H", "12H", "24H", "MID"
+	tickerTimeLayout   = "20060102150405"
+	tradeTimeLayout    = "2006-01-02 15:04:05.000000"
+	location           *time.Location
+	errWsSubFailure    = errors.New("subscription failure")
```

Comment:
This looks like it would be better as an error in the `stream` package and usable for `websocket.go:213` as there isn't one for it.